docs/custom-domain/registry-setup.md

```diff
@@ -0,0 +1,82 @@
+# Using Custom Domains with Azure Container Registry
+
+Every ACR is accessed using its login server. If you have a registry called `myregistry`, you access it using its default hostname, `myregistry.azurecr.io` (in Azure Public Cloud.) As a customer belonging to an organization, you may prefer to access your registry using a custom domain that is associated with your organization, for instance, `container-registry.contoso.com`.
+
+The following steps describe how you can achieve this.
+
+## Prerequisites
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest), [curl](https://curl.haxx.se/) (or a similar tool)
+  - Consider using [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+- An Azure Container Registry. See [here](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli) for instructions on how to create one.
+- Your custom domain names. The following two are required:
+  - custom registry domain to access the registry REST API: `container-registry.contoso.com` 
+  - custom data domain to access the registry content: `eastus-registry-data.contoso.com`
+    - Note that the custom data domain is region specific. For geo-replicated registries, each region should have its own custom data endpoint.
+
+  For each domain, you must prepare a single PEM formatted file containing the TLS private key and public certificate:
```

Comment:
Do you have openssl commands to create the certificate?

Reply:
Added an example